tests/test_cases/issue_637/issue_637.py

```diff
@@ -0,0 +1,18 @@
+import cocotb
+from cocotb.triggers import Timer, NullTrigger
+
+
+@cocotb.test()
+def test_stack_overflow(dut):
```

Comment:
Can we put this in `test_scheduler`?

Reply:
Or perhaps `NullTrigger` can be thought of a concurrency primitive, since I think it lets another coroutine be scheduled?

Reply:
It was made in reaction to a scheduler bug, so `test_scheduler` makes sense.